concordium-consensus/src/Concordium/GlobalState/Block.hs

```diff
@@ -330,15 +330,18 @@ instance BlockData (Block pv) where
     {-# INLINE blockTransactions #-}
 
 instance (IsProtocolVersion pv) => EncodeBlock pv (Block pv) where
-    putBlock _ (GenesisBlock gd) = put genesisSlot >> put gd
+    putBlock _ (GenesisBlock gd) = put genesisSlot >> (putGenesisConfiguration gd)
```

Comment:
It is not clear to me that `getGenesisConfiguration` is and always will be the inverse of `putGenesisConfiguration`. This seems precarious and at the very least requires documentation.

Reply:
It is documented as part of `getGenesisConfiguration`.